ddtrace/internal/flare/_subscribers.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime
```

Comment:
If it feels more appropriate to have this in the same file as the other RC stuff (`ddtrace/internal/remoteconfig/_subscribers.py`) I can move it there. I originally had it there, but then realized this would ping the remoteconfig approver group each time. Wasn't sure if that was necessary.

Reply:
I think it makes sense to live in this file.